recipes/libyaml/all/conanfile.py

```diff
@@ -0,0 +1,68 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibYAMLConan(ConanFile):
+    name = "libyaml"
+    description = "LibYAML is a YAML parser and emitter library."
+    topics = ("conan", "libyaml", "yaml", "parser", "emitter")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/yaml/libyaml"
+    license = "MIT"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = '{}-{}'.format(self.name, self.version)
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["BUILD_TESTING"] = False
```

Comment:
```suggestion
```

not required, it's False by default: https://github.com/yaml/libyaml/blob/0.2.5/CMakeLists.txt#L102